projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -869,19 +847,79 @@ public static function blocks_variation() {
 		// Default to production blocks.
 		$block_varation = 'production';
 
-		if ( Constants::is_true( 'JETPACK_BETA_BLOCKS' ) ) {
+		/*
```

Comment:
This is all a bit more complex than I am really happy, but I kept in mind that this should become simpler in a few months once we can remove the deprecated things and the constants. The way this is currently implemented, I think it should be simple for anyone to remove what's not needed anymore.